### ISSUE TITLE
why results of demo.py and test_net.py are different?

### ISSUE THREAD

**Issue by xsliu2018** [jwyang/faster-rcnn.pytorch#811]
I trained a model on my own dataset and run demo.py to predict a picture, but I got no detection. I use the same picture to run test_net.py,and got a perfect prediction. I got 0.901 mAP on the test set.

**Comment by GSteinberg** [comment]
Did you find out the answer to this question? I also found that the coordinates and the scores predicted in test_net.py and demo.py are different.

**Comment by xsliu2018** [comment]
> Did you find out the answer to this question? I also found that the coordinates and the scores predicted in test_net.py and demo.py are different.

Do you modified [lib/model/utils/config.py](https://github.com/jwyang/faster-rcnn.pytorch/blob/master/lib/model/utils/config.py)? Try to check whether your config is different when running demo.py and test_net.py
